addons/hr_holidays/views/hr_leave_views.xml

```diff
@@ -644,7 +645,7 @@
         <field name="search_view_id" ref="hr_leave_view_search_my"/>
         <field name="view_ids" eval="[(5, 0, 0),
                 (0, 0, {'view_mode': 'kanban', 'view_id': ref('hr_leave_view_kanban')})]"/>
-        <field name="domain">[('user_id', '=', uid), ('employee_company_id', 'in', allowed_company_ids)]</field>
+        <field name="domain">[('user_id', '=', uid)]</field>
```

Comment:
It's part of another task I rebased this branch on. It basically adds this as a multicompany rule, so that it doesn't have to be added every single time. It was causing a bug in this branch.
https://www.odoo.com/web#id=3383478&cids=1&menu_id=4720&action=333&active_id=1251&model=project.task&view_type=form